src/test/regress/expected/multi_table_ddl.out

```diff
@@ -66,7 +64,5 @@ SELECT * FROM pg_dist_shard_placement;
 -- before creating the extension to expire extension specific variables which
 -- are cached for performance.
```

Comment:
We should be able to remove the reconnect now, and do some basic test (create distributed table, do a query)